src/ui/components/AwaitApis.tsx

```diff
@@ -20,9 +21,13 @@ export function AwaitApis({ children }: HTMLAttributes<HTMLDivElement>): React.R
     status !== 'READY' && setMessage(`Connecting to ${endpoint}...`);
   }, [db, keyringStatus, status, endpoint]);
 
-  return (
-    <>
-      {error ? <ConnectionError /> : isLoading ? <Loader message={message} isLoading /> : children}
-    </>
-  );
+  if (keyringStatus === 'READY' && keyring?.getAccounts().length === 0) {
```

Comment:
I think we should display 2 separate messages for:
1. Extension not installed
2. Extension installed but no accounts

Right now it's a bit confusing that after installing the extension I get the same message.